eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -101,6 +101,9 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('security.yml');
         // Slots
         $loader->load('slot.yml');
+        if (interface_exists('FOS\JsRoutingBundle\Extractor\ExposedRoutesExtractorInterface')) {
```

Comment:
@bdunogier @lolautruche @emodric actually, it seems like this *last minute* change broke the patch `$container->hasExtension('fos_js_routing')` is not true (`$container->getExtensions()` returns an empty array at that stage) while with `interface_exists` this works well. So unless, there's an alternative, I'm going to restore the previous condition.